### PR TITLE
doc: enrich the document for /examples

### LINKED ISSUE #89: [Documentation/user guide] How to write wasm scheduler plugin
/kind documentation

Document how to write wasm scheduler plugins with /examples.
This document would be for users, and no need to deep dive into the details that users don't need to know.

### PR DESCRIPTION
**Pull request by sanposhiho**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Enrich the document for /examples for a better visibility to users.

#### Which issue(s) this PR fixes:

Fixes #89

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### What are the benchmark results of this change?

```benchstat

```

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Examples
+
+This examples help you understand how you build your wasm plugins with our Go SDK (which is the only language SDK supported for now).
```
Comment (Gekko0114) [inline examples/README.md:3]: nit

```suggestion
These examples help you understand how you build your wasm plugins with our Go SDK (which is the only language SDK supported for now).
```

**Review by Gekko0114 — APPROVED** [review]
I left one minor comment. Overall looks good.

**Comment by Gekko0114** [comment]
Leave lgtm label to @utam0k

**Comment by utam0k** [comment]
/lgtm

Cool

File: examples/README.md (hunk 1 above)
Comment (utam0k) [reply examples/README.md]: Fixed https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/pull/96/commits/8a9331d771913e32cc7cedcad22dd238fef04c06

**Pull request merged**